scripts/secnetperf.ps1

```diff
@@ -5,8 +5,17 @@ NOTE:
 This script assumes the latest MsQuic commit is built and downloaded as artifacts in the current session.
 This should have been accomplished in netperf/quic.yml.
 
+.PARAMETER Logging
+    If true, will enable logging for the test. This will create a log file in the artifacts directory.
+    Default is false.
+
 #>
 
+param (
+    [Parameter(Mandatory = $false, ParameterSetName='Logging')]
+    [switch]$Logging = $false,
```

Comment:
It should be a `String` (default to empty) to specify the logging profile instead.
```suggestion
    [ValidateSet("", "Basic.Light", "Datapath.Light", "Datapath.Verbose", "Stacks.Light", "Stacks.Verbose", "RPS.Light", "RPS.Verbose", "Performance.Light", "Basic.Verbose", "Performance.Light", "Performance.Verbose", "Full.Light", "Full.Verbose", "SpinQuic.Light", "SpinQuicWarnings.Light")]
    [string]$LogProfile = "",
```